mode/htmlmixed/htmlmixed.js

```diff
@@ -91,10 +91,11 @@
     function html(stream, state) {
       var tagName = state.htmlState.tagName;
       var tagInfo = tagName && tags[tagName.toLowerCase()];
+      var isTagInfoArray = Object.prototype.toString.call(tagInfo) === "[object Array]";
```

Comment:
I also considered using Array.isArray, but IE8 doesn't support that.  I took this from the polyfill at https://developer.mozilla.org/ro/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray.